documentation/CAMARA-Security-Interoperability.md

```diff
@@ -90,6 +91,18 @@ The following table defines the REQUIRED behaviour of the API Provider for the `
 
 The OIDC Authorization Code Flow is defined in [OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html)
 
+### Signed Authentication Requests
+
+It is RECOMMENDED that signed authentication requests be used, as specified by [OIDC](https://openid.net/specs/openid-connect-core-1_0.html#JWTRequests). The same key MAY be used for signing the authentication request as is used for [client authentication](#client-authentication). 
```

Comment:
```suggestion
It is RECOMMENDED that signed authentication requests be used, as specified by [RFC9101](https://www.rfc-editor.org/rfc/rfc9101.html). The same key MAY be used for signing the authentication request as is used for [client authentication](#client-authentication). 
```

Reply:
A text like "as specified by [RFC9101](https://www.rfc-editor.org/rfc/rfc9101.html" would **force** readers and developers to read RFC9191. I would like to avoid that because OIDC says that:
> Note that the Request Objects defined here are compatible with those specified by [The OAuth 2.0 Authorization Framework: JWT-Secured Authorization Request (JAR)](https://openid.net/specs/openid-connect-core-1_0.html#RFC9101) [RFC9101].

How about adding a note that the security considerations of rfc9101 apply?

Reply:
I edited the PR so that it now states that the Security Considerations of RFC9101 apply. I think that with that text the profile does not force developers to read RFC9101 but they can trust this document that we considered all security advise.

Reply:
You made a counter proposal, and without giving me a chance to respond, you made a new commit that changed the text again. Could we please just leave the text as it was and make the reference to RFC9101 just for the sake of reaching a final agreement? We at Telefónica have already accepted the rest of the suggestions made by both @mhfoo and you.

Reply:
You can also respond and have all the chances you want. Please comment make a new review change request. Noting I proposed is written in stone. 
Yes, I made a "counter proposal" and I think making proposals is the fastest way forward. I added my coment and then proposed new text. I think this procodure allows us to move forward fastest.

I think review-change requests and adaption to the PR give us a way to come to an agreement much faster than comments in issues. Please trust me that I do not want to force that new text on you or anybody. 
Please provide a new review-comment or change request. This is your chance.

Reply:
OK, I see. It's as if I was directly committing my change rather than making it a change suggestion... but ok, I'll let this go anyway.